apps/gateway-ui/src/components/WithdrawCard.tsx

```diff
@@ -111,9 +114,11 @@ export const WithdrawCard = React.memo(function WithdrawCard({
             <NumberInput
               min={0}
               max={balanceMsat / 1000}
-              value={amount}
+              // FIXME: we should subtract expected fees here in withdrawAll case
+              value={withdrawAll ? balanceMsat / 1000 : amount}
```

Comment:
This shows your whole balance, and should instead show `balance - expected_fees`